com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/CreateOrUpdateServiceBrokerStep.java

```diff
@@ -155,8 +157,12 @@ protected void updateServiceBroker(DelegateExecution context, CloudServiceBroker
                         getStepLogger().warn(Messages.UPDATE_OF_SERVICE_BROKERS_FAILED_403, serviceBroker.getName());
                         return;
                     }
+                    getStepLogger().error(Messages.ERROR_CREATING_SERVICE_BROKERS, e, ExceptionMessageTailMapper
```

Comment:
We should avoid call computeTail in steps.
Instead we have to override the method getStepErrorMessage with extended error message. 

Also I am not completely sure if this is true when registering space scoped service brokers, who are owned by the MTA, to  to mapped to specific support ticket component.
For instance, I declare my "boyan-broker" and its registering fails for some reason. What would be the error message? I believe the error message should not point to any service broker component at all.